tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -276,3 +308,13 @@ def _a11y_click_menu_item(self, menu_item_locator):
     def _wait_for_toggle_ready(self, by, locator):
         checkbox = self.marionette.find_element(by, locator)
         Wait(self.marionette).until(expected.element_enabled(checkbox))
+
+    # this method is a copy of go_back() method in regions/keyboard.py
+    def return_to_main_menu(self, gaia_header=True):
```

Comment:
Not sure if we need to specify `gaia_header` in the test. @npark-mozilla, do you think it's a good idea to try to search for the regular gaia_header and if it's not present, find the non gaia one?